recipes/xmitgcm/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xmitgcm" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4378d4982a86786daaf49690ad3d144a1296c2fa385ed86f03394e5b655ea8f9
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray
```

Comment:
In the `setup.py`, there are version restrictions on `xarray` and `dask`? Should they be included?

```
INSTALL_REQUIRES = ['xarray >= 0.10.1', 'dask >= 0.12']
```

Reply:
fixed